pkgs/applications/virtualization/arion/arion.nix

```diff
@@ -0,0 +1,50 @@
+{ pkgs ? import ./. {}
+, arion-compose
+, arion-compose-eval-src
+}:
```

Comment:
The same thing here about turning this into a `callPackage`-style package.

The current way it is written, it is not obvious that `arion` depends on things like `justStaticExecutables`, `overrideCabal`, `docker-compose`, `makeWrapper`, etc.  It would also be more difficult for someone to override those arguments, since they are taken directly from `pkgs`.